e2eTests/README.md

```diff
@@ -6,23 +6,8 @@ of how this setup works.
 
 ![FEGA Norway E2E Test Setup Module](figure-1.png)
 
-[Edit and export this figure at tldraw.com](https://www.tldraw.com/r/hQuNVXYht2-H6QRZcMh28?v=-3234,-969,4361,2023&p=page)
-
-# Prerequisites
-
-1. [Install `docker`](https://docs.docker.com/engine/install/ubuntu/).
-   - You must be able to run `docker` and `docker compose` without sudo
-2. [Install `docker compose`](https://docs.docker.com/compose/install/) (usually it is bundled with Docker).
-3. [Install Golang](https://go.dev/doc/install)
-   - `curl -OJL https://go.dev/dl/go1.22.2.linux-amd64.tar.gz`
-   - `rm -rf /usr/local/go && tar -C /usr/local -xzf go1.22.2.linux-amd64.tar.gz`
-   - `export PATH=$PATH:/usr/local/go/bin`
-4. [Install Java](https://www.java.com/en/download/help/download_options.html)
-   - `sudo apt update`
-   - `sudo apt install openjdk-21-jdk`
-   - `echo "export JAVA_HOME=/usr/lib/jvm/java-21-openjdk-amd64/" >> ~/.bashrc`
```

Comment:
This wouldn't work if someone is using other shells.

Reply:
Hei, it was just an example for them to show that `JAVA_HOME` must be in the environment.

I could add a comment here saying it should be specific for their shell env. WDYT?

Reply:
That would be more appropriate yes :)

Reply:
Addressed in: [84821c9](https://github.com/ELIXIR-NO/FEGA-Norway/pull/148/commits/84821c91d4f989235311077f65b44b828d77b4db)